dojo/tools/rusty_hog/parser.py

```diff
@@ -0,0 +1,125 @@
+import json
+from dojo.tools.parser_test import ParserTest
+from dojo.models import Finding
+
+
+class RustyhogParser(object):
+
+    def get_scan_types(self):
+        return ["Rusty Hog Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type  # no custom label for now
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Rusty Hog Scan - JSON Report"
+
+    def get_findings(self, json_output, test):
+        tree = json.load(json_output)
+        return self.get_items(tree, test)
+
+    def parse_json(self, json_output):
+        try:
+            data = json_output.read()
+            try:
+                tree = json.loads(str(data, 'utf-8'))
+            except:
+                tree = json.loads(data)
+        except:
+            raise Exception("Invalid format")
+        return tree
```

Comment:
This does not appear to be used. Can you either use it in places where `json.load()` is used?